CODE_OF_CONDUCT.md

```diff
@@ -53,7 +53,7 @@ further defined and clarified by project maintainers.
 ## Enforcement
 
 Instances of abusive, harassing, or otherwise unacceptable behavior may be
-reported by contacting the project team at [dotnet@sonarsource.com]. All
+reported by contacting the project team at [Community Forum](https://community.sonarsource.com/). All

```

Comment:
hmmm indeed I don't know about this one...
could we just align with Sonar Java regarding this? https://github.com/SonarSource/sonar-java/blob/master/CONTRIBUTING.md

do we need this?

Reply:
Do we need to replace dotnet@? I think yes. This PR is baby step to address this.

Do we need Code of Conduct itself? I don't know. It sounds like a legal question to me => out of scope of this issue.
I personally don't see a problem if we drop the whole file. But there might be a reason why it was added.